src/libraries/System.Formats.Cbor/tests/Reader/CborReaderTests.cs

```diff
@@ -54,6 +54,79 @@ public static void Depth_ShouldReturnExpectedValue(int depth)
             Assert.Equal(CborReaderState.Finished, reader.PeekState());
         }
 
+        [Fact]
+        public static void Reset_ConformanceMode()
+        {
+            var reader = new CborReader(Array.Empty<byte>());
+            Assert.Equal(CborConformanceMode.Strict, reader.ConformanceMode);
+
+            reader.Reset(Array.Empty<byte>());
+            Assert.Equal(CborConformanceMode.Strict, reader.ConformanceMode);
+
+            foreach(var mode in new[] { CborConformanceMode.Canonical, CborConformanceMode.Ctap2Canonical, CborConformanceMode.Lax, CborConformanceMode.Strict })
+            {
+                reader = new CborReader(Array.Empty<byte>(), mode);
+                Assert.Equal(mode, reader.ConformanceMode);
+
+                reader.Reset(Array.Empty<byte>());
+                Assert.Equal(mode, reader.ConformanceMode);
+            }
+        }
+
+        [Fact]
+        public static void Reset_AllowMultipleRootLevelValues()
```

Comment:
```suggestion
        public static void Reset_DoesNotAffect_AllowMultipleRootLevelValues()
```

Reply:
Similar to my feedback https://github.com/dotnet/runtime/pull/88104/files#r1254625862, do these tests suggest that the `Reset` method should be accepting the constructor parameters as optional parameters?